libs/langgraph/langgraph/utils/runnable.py

```diff
@@ -78,9 +77,19 @@ class StrEnum(str, enum.Enum):
     ),
     (
         sys.intern("store"),
-        (BaseStore, "BaseStore", inspect.Parameter.empty),
+        (
+            BaseStore,
+            "BaseStore",
+            Optional[BaseStore],
+            # Best effort to catch some forward references.
+            # This will not work for cases like `"Union[None, BaseStore]"`,
+            # we'll need to re-write logic to use get_type_hints()
+            # to resolve forward references.
+            "Optional[BaseStore]",
+            inspect.Parameter.empty,
+        ),
         CONFIG_KEY_STORE,
-        inspect.Parameter.empty,
+        None,
```

Comment:
Actually this will introduce a new bug since `None` shouldn't be assigned to BaseStore -- i'll split this into two cases prior to merging